Dockerfiles/manifests/openshift/tf-cluster/variables.tf

```diff
@@ -0,0 +1,11 @@
+//  The region we will deploy our cluster into.
+variable "region" {
+  description = "Region to deploy the cluster into"
+  default = "us-east-2"
+}
+
+//  The public key to use for SSH access.
+variable "public_key_path" {
```

Comment:
lets not use the developer public key, but rather generate a new pair
check `aws-eks/tf-cluster`

Reply:
changed to a newly generated keypair, updated readme to include the command to generate keypair